### PR TITLE
working coil helper

### LINKED ISSUE #47: Help coil calculator
coil_help

### PR DESCRIPTION
**Pull request by gbaranski**
done :) @stasbar 

image

closes #47

### CODE REVIEW

**Comment by gbaranski** [comment]
@stasbar pls merge

**Comment by stasbar** [comment]
closed in #47

**Comment by gbaranski** [comment]
@stasbar can u merge?

**Comment by stasbar** [comment]
Ok, I need to test it first

**Comment by stasbar** [comment]
@gbaranski you forgot about Coil type and Wires Section. Check out https://github.com/vape-tool/VapeTool-Webapp/issues/47#issuecomment-640069343

File: src/pages/coil/CoilCalculator.tsx [hunk 1]
```diff
@@ -250,7 +263,37 @@ const CoilCalculator: React.FC<CoilCalculatorProps> = props => {
...
+            }
+          >
+            Next
+          </Button>,
+        ]}
+        // onOk={this.handleOk}
+        // onCancel={this.handleCancel}
+      >
+        <CoilHelper setSlider={setSlider} />
+      </Modal>
```
Comment (stasbar) [inline src/pages/coil/CoilCalculator.tsx:295]: Move whole element Modal to CoilHelper.

File: src/pages/coil/CoilCalculator.tsx [hunk 2]
```diff
@@ -250,7 +263,37 @@ const CoilCalculator: React.FC<CoilCalculatorProps> = props => {
...
+            key="next"
+            type="primary"
+            onClick={() =>
+              slider ? slider.next() : console.error('Please restart page, something went wrong')
+            }
+          >
+            Next
+          </Button>,
+        ]}
+        // onOk={this.handleOk}
```
Comment (stasbar) [inline src/pages/coil/CoilCalculator.tsx:291]: remove unused code

File: src/components/CoilHelper/index.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      touchMove
+      ref={c => props.setSlider(c)}
+    >
+      <div>
+        <div style={{ height: '14em' }}>
+          <h1>Inner coil diameter</h1>
+          <h3>This is diameter of drill on which you are wrapping your coil</h3>
+        </div>
+        <ImageWebp
+          png={innerDiameterPng}
```
Comment (stasbar) [inline src/components/CoilHelper/index.tsx:25]: you can use jpg photo in the `png` parameter 😄

File: src/components/CoilHelper/index.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Carousel } from 'antd';
+import ImageWebp from '../ImageWebp';
+import innerDiameterPng from '../../assets/innerCoilDiameter.png';
```
Comment (stasbar) [inline src/components/CoilHelper/index.tsx:4]: you can use jpg photos. ImageWebp accept jpg in the png parameter 😄

**Pull request merged**